tests/integration/services/storage/test_redis_integration.py

```
@@ -0,0 +1,580 @@
+"""Integration tests for RedisClient with real Redis instance."""
+
+import json
+import time
+from typing import Any, Generator
```

Ruff (0.14.0)

Import from `collections.abc` instead: `Generator`

Import from `collections.abc`

(UP035)